### ISSUE TITLE
No internet connection upon startup, only after minutes of waiting

### ISSUE THREAD

**Issue by frukbachers** [StevenBlack/hosts#2064]
I have just replaced my hosts file with the unified only hosts file in the specified location on WIN11 and initially it worked fine. 

Upon a restart I noticed that internet was non-functional upon startup, only after a few minutes of waiting did it connect to the internet.

Is this caused by inproper installation/setup on my end or could there be another issue?

**Comment by StevenBlack** [comment]
Hi @frukbachers thank you for your observation. This seems to come up fairly often here.  

This has to do with Windows DNS. Try disabling the DNS Client Service. Does this resolve it for you?

This isn't caused by improper installation or setup. It's Windows. 😄

**Comment by frukbachers** [comment]
I always assume incompetence on my end first when setting up something new.

I disabled the DNS client service (dnscache) through regedit (services.msc yielded a greyed out context and properties window for that service) and after a restart internet works completely fine. I didn't see any banned ads in my apps just yet and my net upon startup is back to being available.

Thanks for the quick reply.
